apiary.apib

```diff
@@ -407,34 +392,63 @@ Lists all available regions.  The id returned can be used in conjunction with `/
             Accept-Encoding: identity
 
     + Attributes
-        + message: `Example: The gift message!` (string, optional) - Gift email message. Required if no supplied template.
-        + subject: `Example: Please enjoy this gift!` (string, optional) - Gift email subject. Required if no supplied template.
-        + `gift_template`: `Example: ABCDE` (string, optional) - The template id used for this campaign, as per the template section of your web account.  Recommended, but not required if providing the `message` and `subject` as part of the create campaign API call.
+        + message: `Example: The reward message!` (string, optional) - Reward email message. Required if no supplied template.
+        + subject: `Example: Please enjoy this reward!` (string, optional) - Reward email subject. Required if no supplied template.
+        + `gift_template`: `Example: ABCDE` (string, optional) - The template id used for this order, as per the template section of your web account.  Recommended, but not required if providing the `message` and `subject` as part of the create order API call.
         + `contacts` (array[Contact], required) - A list of contact objects and is required.
-        + `price_in_cents`: `2500` (number, required) - the value of the offer you are sending, in cents.  A value of 2500 would send a $25 gift offer for the given brand(s)
-        + `brand_codes`: `Example:["itunesus","amazonus"]` (array[string], required) - a list of one or more brand_code as given by the `/brands` endpoint to send in your campaign.  To give your recipients a choice, simply provide more than one brand_code; recipients will then get to choose the brand they prefer as part of the claim process.
-        + expiry: `Example: YYYY-MM-dd` (string, required) - The campaign's claim before date. After the end of this date (in Pacific Standard Time), recipients in this campaign will not be able to redeem their gift offers and the appropriate portion of the credits for any unclaimed gift offers will be returned to your account.  It may not exceed one year from the time of creation.
-        + id: `Example: abc123` (string, required) - An identifier for the campaign assigned by the client (you).  Must be unique across all your campaigns.  If an unexpected error occurs sending your campaign, you can safely send the same POST again with the same `id`.
+        + `price_in_cents`: `2500` (number, required) - the value of the offer you are sending, in cents.  A value of 2500 would send a $25 reward offer for the given brand(s)
+        + `brand_codes`: `Example:["itunesus","amazonus"]` (array[string], optional) - custom list of one or more brand_code as given by the `/brands` endpoint to send in your order. Use this to limit choice to a few select brands.
+        + `region`: `Example: CA` (string, optional) - ISO 3166-2 region code to select the Full Catalog region recipients will get to choose a brand from as a part of the claim process.
```

Comment:
Unless this has been updated, the expected input is the full region name (eg. "Canada" or "USA"), not the ISO region code.

Reply:
Adjusted based on that, part of the docs was mentioning the wrong value as well.